src/main/webapp/javascript/event.js

```diff
@@ -0,0 +1,137 @@
+var id = getQueryVariable("id");
+
+function loadEvent() {
+    fetch('/api/event/' + id, {
+        "method": "GET",
+        headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json'
+        }
+    }).then(function (response) {
+        return response.json();
+    }).then(function (event) {
+        console.log(JSON.stringify(event));
+        if (event !== undefined) {
+            document.getElementById("event-field").classList.remove("w3-hide");
+            w3.displayObject("title", event);
+            w3.displayObject("event-field", event);
+        }
+        if (event.eventStatus === "OPEN") {
```

Comment:
This should probably be moved inside the previous `if`, otherwise this may not work if `event` is undefined.